tomviz/CentralWidget.cxx

```diff
@@ -146,6 +157,7 @@ void PopulateHistogram(vtkImageData *input, vtkTable *output)
   output->AddColumn(populations.GetPointer());
 }
 
+//-----------------------------------------------------------------------------
```

Comment:
In general we haven't been using these, and I would prefer not to add them. In a modern code editor I don't think they add much, but don't worry about removing them. I will probably do a consistency sweep soon...